apps/builder/app/builder/features/settings-panel/controls/boolean.tsx

```diff
@@ -23,7 +24,7 @@ export const BooleanControl = ({
   onChange,
   onDelete,
 }: ControlProps<"boolean">) => {
-  const label = humanizeString(meta.label || propName);
+  const label = humanizeString(meta.label || propName, humanizeAttribute);
```

Comment:
can it be just humanizeAttribute using humanizeString instead of customHumanize which is invoked by humanizeString?

Reply:
like 
```js
const label = humanizeAttribute(meta.label || propName);
```